content/index.html

```diff
@@ -108,6 +114,8 @@
       <div class="col-md-2 col-sm-4 col-xs-6 logos">
         <a href="https://www.variomedia.de/"><img src="assets/company-logos/variomedia.png"/></a>
       </div>
-    </div>
+      <div class="col-md-2 col-sm-4 col-xs-6 logos">
+        <a href="https://www.weave.works/"><img src="assets/company-logos/weave.png"/></a>
```

Comment:
I found this one: http://www.ricston.com/wp-content/uploads/2015/07/P-weaveWorks_colour_logo_POS_RGB_320x132.png

We could certainly asked them for one, but the size seems sufficient.